### ISSUE TITLE
Simple tagged template configuration no longer working.

### ISSUE THREAD

**Issue by stolksdorf** [Thom1729/Sublime-JS-Custom#112]
As of this morning my JSCustom configuration stopped working properly. This is what the syntax highlighting is looking like now

![image (2)](https://user-images.githubusercontent.com/387742/117366654-1a50ee80-ae8f-11eb-8c37-2b78f2966d24.png)

this is the configuration I am using:

```
{
    "configurations": {
        "Javascript Extended": {
            "custom_templates": {
                "tags": {
                    "style": "scope:source.css",
                    "css": "scope:source.less",
                    "x": "scope:text.html.basic"
                }
            }
        }
    }
}
```

When I check the console I am getting the following errors: 
![image (3)](https://user-images.githubusercontent.com/387742/117366806-4c625080-ae8f-11eb-9ee0-83f3856d3c96.png)

I also had a friend try a complete fresh install of Sublime Text 3 (portable on windows 10) with JSCustom and my configuration and had similar results:

With a similar console. 

![image](https://user-images.githubusercontent.com/387742/117367034-9a775400-ae8f-11eb-925a-bdb9ef8b46b8.png)

I've been using this configuration for several months with no issue. I did not update or change anything. I tried manually installing v3.0.0 and had similar issues. I've tried completing removing the package and re-installing it and had the same issue as well. 

Any help is greatly appreciated.

**Comment by Thom1729** [comment]
Thanks for the report and for the error log. This should be fixed in 3.0.5 and in 4.0.3. There was an internal change to path handling that seems to be breaking in certain situations. I've undone that change.

**Comment by stolksdorf** [comment]
Just installed `v3.0.5` and everything is working perfectly again. Thank you so much for responding so quickly and effectively. Just incredible stuff, you made my day. I hope you have a lovely weekend!
